modules/gapi/perf/streaming/gapi_streaming_source_perf_tests.cpp

```diff
@@ -205,14 +205,15 @@ PERF_TEST_P_(OneVPLSourcePerf_PP_Engine_Test, TestPerformance)
     }
     VPPPreprocEngine preproc_engine(std::move(policy));
     cv::gapi::wip::Data out;
+    cv::util::optional<cv::Rect> empty_roi;
     TEST_CYCLE()
     {
         source_ptr->pull(out);
         cv::MediaFrame frame = cv::util::get<cv::MediaFrame>(out);
         cv::util::optional<pp_params> param = preproc_engine.is_applicable(frame);
         pp_session sess = preproc_engine.initialize_preproc(param.value(),
                                                             required_frame_param);
-        (void)preproc_engine.run_sync(sess, frame);
+        (void)preproc_engine.run_sync(sess, frame, empty_roi);
```

Comment:
Honestly, I didn't understand how it works
Does it do `crop` if `roi` is passed and doesn't do `crop` otherwise.